src/components/BBAnimated/BBAnimated.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react'
+import './BBAnimated.scss'
+
```

Comment:
I'm definitely going to credit the author in the ReadMe, but would be be ok to add some commented out code crediting the author for this below the imports?  It's what I did on my last project and Brittany seemed to think it was enough

Reply:
Nevermind!  I see it in the scss file.

Reply:
Yes, in scss was first author already, and I just added author who created it for React